### PR TITLE
Log/fetch version

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@matthiasrohmer still not ready, will ping you

**Comment by sebastianbenz** [comment]
@matthiasrohmer this is now ready for review

File: platform/lib/routers/runtimeLog.js [hunk 1]
```diff
@@ -15,33 +15,45 @@
  */
 
 'use strict';
+
 const express = require('express');
-const LogFormatter = require('../runtime-log/formatter');
+const LogFormatter = require('../runtime-log/HtmlFormatter.js');
 
-const Templates = require('@lib/templates/');
+const Templates = require('../templates/');
```
Comment (matthiasrohmer) [inline platform/lib/routers/runtimeLog.js]: Just curious: is there a reason you've been moving away from module-register syntax (`@lib/`)?

Comment (sebastianbenz) [reply platform/lib/routers/runtimeLog.js]: Jest doesn't support it....

**Pull request merged**